Gopkg.lock

```diff
@@ -102,6 +102,7 @@
   digest = "1:3dd078fda7500c341bc26cfbc6c6a34614f295a2457149fc1045cab767cbcf18"
```

Comment:
Did Gopkg.toml also need to be modified?

Reply:
It gets updated by "dep" when the set of input files changes.

Reply:
Looks like we now have a conflict because of that. I can rebase if this approach is how we want to continue.